<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressDialog</class>
 <widget class="QDialog" name="ProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="load_progress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="preprocess_progres">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="load_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading</string>
   </property>
  </widget>
  <widget class="QLabel" name="preprocess_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>67</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Preprocessing</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="feature_ext_progres">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="feature_ext_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Feature extraction</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="cluster_progress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="cluster_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>167</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clustering</string>
   </property>
  </widget>
  <widget class="QLabel" name="dimRedox_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Dimensionality reduction</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="dimRedox_progress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
